metron-deployment/amazon-ec2/ansible.cfg

```diff
@@ -25,4 +25,4 @@ log_path = ./ansible.log
 
 # fix for "ssh throws 'unix domain socket too long' " problem
 [ssh_connection]
-control_path = %(directory)s/%%h-%%p-%%r
+control_path = /tmp/%%h-%%p-%%r
```

Comment:
I wouldn't use temp. DoD stigs require noexec on tmp. Alternatively, see if the socket will work when /tmp is mounted noexec.